llvm/lib/ProfileData/MemProf.cpp

```diff
@@ -499,12 +500,56 @@ void CallStackRadixTreeBuilder::build(
 
   // Sort the list of call stacks in the dictionary order to maximize the length
```

Comment:
Should this comment be removed?

Reply:
Fixed in the latest iteration.  I've removed the old comment and rephrased the immediately following paragraph.